CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 # `wasm-bindgen` Change Log
 --------------------------------------------------------------------------------
 
+## Unreleased
+
+### Changed
+
+* Remove unnecessary JSDoc annotations from generated `.d.ts` files
```

Comment:
```suggestion
* Remove unnecessary JSDoc type annotations from generated `.d.ts` files
```
Is that the right wording, or is it more confusing?

Reply:
No you're right. There could be user-defined JSDoc annotations and those aren't removed.